projects/boost/build.sh

```diff
@@ -16,7 +16,9 @@
 ################################################################################
 
 # Build boost
-./bootstrap.sh && ./b2 headers
+CXXFLAGS="-stdlib=libc++ -pthread" LDFLAGS="-stdlib=libc++" \
```

Comment:
Did you mean to do CXXFLAGS="$CXXFLAGS ..." ?? I dont think we should remove existing CXXFLAGS.

Reply:
+1, otherwise we're losing coverage and sanitizers instrumentation

Reply:
Done